tests/Python3_Driver_Testing.md

```diff
@@ -96,7 +96,7 @@
 | Kenwood_TH-D72_live_mode |  |  | Yes |
 | Kenwood_TH-D7G |  |  | Yes |
 | Kenwood_TH-F6 |  |  | Yes |
-| Kenwood_TH-F7 |  |  | Yes |
+| Kenwood_TH-F7 | [@sv1](https://github.com/sv1) | 19-Oct-2022 | Yes |
```

Comment:
\o/

Reply:
For this to work, we need the decode() relaxing!